data/data-pipeline/data_pipeline/etl/sources/census/etl.py

```diff
@@ -211,31 +213,35 @@ def _load_national_geojson(self):
         Returns:
             None
         """
-        logger.info("Generating national geojson file")
+        logger.debug("Loading National GeoJson")
 
         usa_df = gpd.GeoDataFrame()
 
         for file_name in self.GEOJSON_BASE_PATH.rglob("*.json"):
-            logger.info(f"Ingesting {file_name}")
+            logger.debug(f"Adding national GeoJSON file {file_name.name}")
             state_gdf = gpd.read_file(file_name)
             usa_df = usa_df.append(state_gdf)
 
         usa_df = usa_df.to_crs(
             "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs"
         )
-        logger.info("Writing national geojson file")
-        usa_df.to_file(self.NATIONAL_TRACT_JSON_PATH, driver="GeoJSON")
 
-        logger.info("Census tract downloading complete")
+        logger.debug("Saving national GeoJSON file")
+        usa_df.to_file(self.NATIONAL_TRACT_JSON_PATH, driver="GeoJSON")
 
     def load(self) -> None:
         """Create state CSVs, National CSV, and National GeoJSON
 
         Returns:
             None
         """
-        logger.info("Saving Census CSV")
+        logger.debug("Loading census data")
+
+        logger.debug("Loading individual state csv files")
         for fips_code in self.TRACT_PER_STATE:
             self._load_into_state_csvs(fips_code)
+
         self._load_national_csv()
         self._load_national_geojson()
+
+        logger.debug("Census data complete")
```

Comment:
Does your goodbye not handle this?

Reply:
Because there are multiple ways the census etl gets run (by itself, or as part of the full run but separate from the other etls) I left that particular debug statement in. I thought it may help in some rando circumstance I've not thought of yet. Happy to take it out if you don't think it'll add any value.

Reply:
nah no strong feeling, and that makes sense. i'd say keep it.